config/node/resources/config-messaging.properties.mustache

```diff
@@ -1,3 +1,5 @@
 [messaging]
 
 subscriberPort = {{{subscriberPort}}}
+
+listenInterface = 0.0.0.0
```

Comment:
move value to shared.yml

Reply:
unsure if this is shared value with the one in config-node or not. If this value could be different, add a prefix to listenInterface like `messaginListenInterface` and `nodeListenInterface`